### PR TITLE
[aws/ecr] Fix roles

### PR DESCRIPTION
**Pull request by goruha**
## What
* Updates ses module

## Why
* Fix https://github.com/cloudposse/terraform-aws-ses-lambda-forwarder/issues/4

### CODE REVIEW

File: aws/ecr/kops_ecr_app.tf [hunk 1]
```diff
@@ -3,14 +3,14 @@ variable "kops_ecr_app_repository_name" {
 }
 
 module "kops_ecr_app" {
-  source       = "git::https://github.com/cloudposse/terraform-aws-kops-ecr.git?ref=tags/0.1.4"
+  source       = "git::https://github.com/cloudposse/terraform-aws-kops-ecr.git?ref=use-roles-instead-of-users"
```
Comment (osterman) [inline aws/ecr/kops_ecr_app.tf]: Pin to release

File: aws/ecr/kops_ecr_user.tf [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module "kops_ecr_user" {
-  source    = "git::https://github.com/cloudposse/terraform-aws-iam-system-user.git?ref=tags/0.3.0"
+  source    = "git::https://github.com/cloudposse/terraform-aws-iam-system-user.git?ref=proxy-role-for-user"
```
Comment (osterman) [inline aws/ecr/kops_ecr_user.tf]: Pin to release

File: aws/ses/main.tf [hunk 3]
```diff
@@ -44,7 +44,7 @@ variable "parent_domain_name" {
 }
 
 module "ses" {
-  source = "git::https://github.com/cloudposse/terraform-aws-ses-lambda-forwarder.git?ref=tags/0.1.0"
+  source = "git::https://github.com/cloudposse/terraform-aws-ses-lambda-forwarder.git?ref=fix-s3-action-index"
```
Comment (osterman) [inline aws/ses/main.tf]: Pin to release

**Comment by goruha** [comment]
use this solution https://github.com/cloudposse/terraform-root-modules/pull/65

**Pull request closed without merging**